tests/support/mocked-liquidprompt.sh

```diff
@@ -0,0 +1,53 @@
+#-------------------------------------------------------------------------------
+# Definitions *PRIOIR* to sourcing liquidprompt
+#...............................................................................
+_lp_connection()
+{
+    echo "$TEST_CONNECTION";
+}
+
+DETACHED_SESSION_COUNT_TMUX=2
+DETACHED_SESSION_COUNT_SCREEN=2
+DETACHED_SESSION_COUNT=4
+
+_tmux="$(which tmux)"
+function tmux
+{
+    if [[ "$1" == list-sessions ]]; then
+        for i in eval "echo {1..$DETACHED_JOB_COUNT_TMUX}"; do
+            echo 'detached'
+        done
+    else
+        "$_tmux" "$@"
+    fi
+}
+
+_screen="$(which screen)"
+function screen
+{
+    if [[ "$1" == -ls ]]; then
+        for i in eval "echo {1..$DETACHED_JOB_COUNT_SCREEN}"; do
+            echo 'Detach)'
+        done
+    else
+        "$_screen" "$@"
+    fi
+}
+
+
+source "$SCRIPT_DIR/liquidprompt"
```

Comment:
A fixed liquidprompt configuration file should be used.
Currently this testsuite fails on my environment because I have a `~/.liquidpromptrc` with non default settings.